core/dbt/tests/util.py

```diff
@@ -91,7 +91,12 @@ def run_dbt(
     if profiles_dir and "--profiles-dir" not in args:
         args.extend(["--profiles-dir", profiles_dir])
     dbt = dbtRunner()
+
+    before_dir = os.getcwd()
     res = dbt.invoke(args)
+    after_dir = os.getcwd()
+    # The directory has not been changed after running dbt command.
+    assert before_dir == after_dir
```

Comment:
It seems like this assertion would be better if it were directly within a test rather than here. Maybe [here](https://github.com/dbt-labs/dbt-core/blob/84eb0ff6720ec82ce4015c2657d512bf51381732/tests/functional/dbt_runner/test_dbt_runner.py#L14) instead?

What do you think @ChenyuLInx?

Reply:
I think this is great!! Also makes it checking all future commands.

Reply:
I added test code to `tests/functional/dbt_runner/test_dbt_runner.py` in this commit.
https://github.com/dbt-labs/dbt-core/pull/9596/commits/26ec953a61bc819e1658457bfc328943a3ff276f